Develop/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+      integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="styles/style.css" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <title>Weather Dashboard</title>
+  </head>
+  <body>
+    <nav class="navbar navbar-dark bg-dark"><h1>Weather Dashboard</h1></nav>
+    <main class="container-fluid">
+      <div class="row content">
+        <div class="col-sm-4 sidenav bg-light">
+          <h4>Search for a city:</h4>
+          <form class="input-group">
+            <input type="text" id="city-input" />
```

Comment:
Pretty much all text inputs and inputs, in general, should have a label for accessibility.